### ISSUE TITLE
Can't use container name/id when using rancher exec

### ISSUE THREAD

**Issue by superseb** [rancher/rancher#12918]
Version: CLI v0.6.9

See below, the regression introduced:

```
[rancher-v0.6.8]# ./rancher ps
ID        TYPE      NAME        IMAGE     STATE     SCALE     SYSTEM    ENDPOINTS   DETAIL
1s31      service   aaa/nginx   nginx     healthy   5/5       false
[rancher-v0.6.8]# ./rancher ps -c
ID        NAME            IMAGE                   STATE     HOST      IP              DOCKER         DETAIL
1i33      aaa-nginx-1     nginx                   running   1h1       10.42.54.252    f10c0708ce86
1i34      aaa-nginx-2     nginx                   running   1h3       10.42.57.28     57fd8071f50e
1i35      aaa-nginx-3     nginx                   running   1h3       10.42.86.220    15cefc3c3403
1i36      aaa-nginx-4     nginx                   running   1h1       10.42.176.183   d24311c47973
1i37      aaa-nginx-5     nginx                   running   1h2       10.42.180.39    4d6241b280c3
1i39      rancher-agent   rancher/agent:v1.2.10   running   1h3                       8f3ab72e50d9
1i40      rancher-agent   rancher/agent:v1.2.10   running   1h1                       8964734600eb
1i41      rancher-agent   rancher/agent:v1.2.10   running   1h2                       ab826e273993
[rancher-v0.6.8]# ./rancher exec -i -t aaa-nginx-1 /bin/bash
root@f10c0708ce86:/# exit
[rancher-v0.6.9]# ./rancher exec -i -t aaa-nginx-1 /bin/bash
FATA[0000] failed to find a container
```

This while `rancher logs aaa-nginx-1` works fine.

Workaround is to use the service name and then select a container
```
[rancher-v0.6.9]# ./rancher exec -i -t aaa/nginx /bin/bash
Containers:
[1] aaa-nginx-1 (10.42.54.252)
[2] aaa-nginx-2 (10.42.57.28)
[3] aaa-nginx-3 (10.42.86.220)
[4] aaa-nginx-4 (10.42.176.183)
[5] aaa-nginx-5 (10.42.180.39)
Select:
```

**Comment by deniseschannon** [comment]
Available with Rancher CLI v0.6.10-rc1.

**Comment by soumyalj** [comment]
Reproduced the issue with CLI v0.6.9
Tested with rancher version v0.6.10-rc1
Exec into the nginx container was successful
```
LAPTOP-6CQEFUBT:rancher-v0.6.10-rc1 soumya$ ./rancher ps -c
ID        NAME                 IMAGE                   STATE     HOST      IP             DOCKER         DETAIL
1i2       rancher-agent        rancher/agent:v1.2.10   running   1h3                      009c23937bab   
1i4       rancher-agent        rancher/agent:v1.2.10   running   1h1                      e364ee3994f3   
1i5       rancher-agent        rancher/agent:v1.2.10   running   1h2                      6a18d3332349   
1i33      Default-service1-1   nginx                   running   1h2       10.42.198.7    1a34b44d46a4   
1i34      Default-service1-2   nginx                   running   1h2       10.42.17.130   7afe4f244030   
LAPTOP-6CQEFUBT:rancher-v0.6.10-rc1 soumya$ ./rancher exec -i -t Default-service1-1 /bin/bash
root@1a34b44d46a4:/# echo "Hello World"
Hello World
```
Exec using the service name and choosing the container to exec was also successful.

```
LAPTOP-6CQEFUBT:rancher-v0.6.10-rc1 soumya$ ./rancher exec -i -t Default/service1 /bin/bash
Containers:
[1] Default-service1-1 (10.42.198.7)
[2] Default-service1-2 (10.42.17.130)
Select: 2
root@7afe4f244030:/# echo "Hello World"
Hello World
root@7afe4f244030:/# exit
exit



```
